libreoffice-core/sw/source/uibase/docvw/edtwin.cxx

```diff
@@ -623,16 +623,19 @@ void SwEditWin::UpdatePointer(const Point &rLPt, sal_uInt16 nModifier )
         }
 
         // which kind of text pointer have we to show - horz / vert - ?
-        if( PointerStyle::Text == eStyle && rSh.IsInVerticalText( &rLPt ))
-            eStyle = PointerStyle::TextVertical;
-        else if (rSh.GetViewOptions()->CanHideWhitespace() &&
-                 rSh.GetLayout()->IsBetweenPages(rLPt))
-        {
-            if (rSh.GetViewOptions()->IsHideWhitespaceMode())
-                eStyle = PointerStyle::ShowWhitespace;
-            else
-                eStyle = PointerStyle::HideWhitespace;
-        }
+        // NOTE: @teo turning off show/hide whitespace between pages
```

Comment:
Can you add in the link to the macro notion ticket do this comment so it's easier to search for by grepping MACRO-

Reply:
done